pkgs/top-level/python-packages.nix

```diff
@@ -6137,8 +6139,22 @@ self: super: with self; {
 
   libcloud = callPackage ../development/python-modules/libcloud { };
 
+  libcomps = lib.pipe pkgs.libcomps [
+    toPythonModule
+    (p: p.overrideAttrs (super: { meta = super.meta // { outputsToInstall = [ "py" ]; }; }))
+    (p: p.override { inherit python; })
+    (p: p.py)
```

Comment:
the `.py` strikes me as yucky, but it is also incredibly creative and does reduce the install size. Note that the `propagatedBuildInputs` go to `dev` and not `py`. Luckily it seems these packages are unaffected

Reply:
it's fine really. i just hadn't seen it before